tools/common/common-build.props

```diff
@@ -11,6 +11,7 @@
     <WindowsTargetPlatformMinVersion Condition="'$(WindowsTargetPlatformMinVersion)'==''">10.0.10586.0</WindowsTargetPlatformMinVersion>
     <EnableDotNetNativeCompatibleProfile Condition="'$(EnableDotNetNativeCompatibleProfile)'==''">true</EnableDotNetNativeCompatibleProfile>
     <IncludeOutputsInPackage>false</IncludeOutputsInPackage>
+    <IncludeFrameworkReferencesInPackage>false</IncludeFrameworkReferencesInPackage>
```

Comment:
(Also tools is a lot easier basically only Logging.dll would be a real issue here since most of the problem is coming from nuproj->nuproj->vcxproj project reference chains.)

@kzu also for FYI. Not sure if the BuildingPackage invoking a unique MSBuild which causes rebuilds would be a bug for Nugetizer in general.